modules/mtc-deploying-a-vm.adoc

```diff
@@ -0,0 +1,77 @@
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtc-deploying-a-vm_{context}"]
+= Deploying a Virtual Machine
+
+When you have installed KubeVirt and CDI, and they are active, create a namespace.
+
+For this example we use the namespace `mig-vm`. And create the following YAML to create a Fedora Virtual Machine.
```

Comment:
Suggestions:

Add .Procedure.
Looks like a single-step procedure, if so, start with an asterisk (*). "Create the following YAML...For this example, we we use..."